orttraining/orttraining/python/training/ortmodule/_custom_op_symbolic_registry.py

```diff
@@ -73,7 +73,7 @@ def nll_loss(g, self, target, weight, reduction, ignore_index):
 @register_symbolic("embedding")
 def embedding(g, weight, indices, padding_idx, scale_grad_by_freq, sparse):
     output = g.op(
-        "org.pytorch.aten::ATen", weight, indices, padding_idx, scale_grad_by_freq, sparse, operator_s="aten::embedding"
+        "org.pytorch.aten::ATen", weight, indices, padding_idx, scale_grad_by_freq, sparse, operator_s="embedding"
```

Comment:
why do we need `org.pytorch.aten` instead of just `aten::op` or even `::op`? Same for `register_gradient`

Reply:
Check https://github.com/microsoft/onnxruntime/pull/10700/files#diff-f5990e7f0828239da8f5c5d49cdf1500670d5225f6b3ebf23526f93ee93870f9R122-R132 for how to simplify namespace for the forward part, at least

Reply:
org.pytorch.aten::ATen is the Op type, not the operator attribute in ATen Op. You can see from the same file that we also register custom Op symbolic by using Ops that are not ATen (for example com.microsoft::SoftmaxCrossEntropyLossInternal), so the domain is required. The sample code you gave is for removing the "aten::" from operator attribute, which is already handled (just check the end of the same line here operator_s="embedding").